src/App.tsx

```diff
@@ -1,173 +1,65 @@
-import { Loader } from './components/Loader';
-
-import './App.scss';
-
-export const App = () => (
-  <div data-cy="app">
-    <nav
-      data-cy="nav"
-      className="navbar is-fixed-top has-shadow"
-      role="navigation"
-      aria-label="main navigation"
-    >
-      <div className="container">
-        <div className="navbar-brand">
-          <a className="navbar-item" href="#/">
-            Home
-          </a>
-
-          <a
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
-        </div>
-      </div>
-    </nav>
-
-    <main className="section">
-      <div className="container">
-        <h1 className="title">Home Page</h1>
-        <h1 className="title">People Page</h1>
-        <h1 className="title">Page not found</h1>
-
-        <div className="block">
-          <div className="box table-container">
-            <Loader />
-
-            <p data-cy="peopleLoadingError" className="has-text-danger">
-              Something went wrong
-            </p>
-
-            <p data-cy="noPeopleMessage">
-              There are no people on the server
-            </p>
-
-            <table
-              data-cy="peopleTable"
-              className="table is-striped is-hoverable is-narrow is-fullwidth"
+import React from 'react';
+import {
+  Route, Routes, NavLink, Navigate, useLocation,
+} from 'react-router-dom';
+import cn from 'classnames';
+import { HomePage } from './components/HomePage';
+import { People } from './components/PeoplePage';
+import { NotFoundPage } from './components/NotFoundPage';
+
+export const App: React.FC = () => {
+  const location = useLocation();
+
+  const isHomePage = location.pathname.endsWith('/home');
```

Comment:
Just for your info, you can replace it with [useMatch](https://reactrouter.com/en/main/hooks/use-match)